cmd/up/up.go

```diff
@@ -221,12 +221,12 @@ func (up *upContext) start(autoDeploy, build bool) error {
 	var err error
 	up.Client, up.RestConfig, err = k8sClient.GetLocalWithContext(up.Dev.Context)
 	if err != nil {
-		kubecfg := config.GetKubeConfigFile()
-		log.Infof("failed to load local Kubeconfig: %s", err)
+		kubecfg := config.GetContextKubeconfigPath()
+		log.Infof("failed to load okteto Kubeconfig: %s", err)
 		if up.Dev.Context == "" {
```

Comment:
if `up.Dev.Context` is defined, we are always using that context, right?

Reply:
if `up.Dev.Context` is defined and you have that context on your `.context/kubeconfig` file it will use that value. This means that you have to run at least one `okteto ctx` against that cluster to configure it and be able to use it

Reply:
if `up.Dev.Context` is defined, and the context accessible, I would use that context even if you never did, for example, `okteto context minikube`